ui/src/testing/stubs/cohorts-api-stub.ts

```diff
@@ -0,0 +1,142 @@
+import {convertToResources, ResourceType} from 'app/utils/resourceActionsReact';
+import {
+  Cohort,
+  CohortsApi,
+  EmptyResponse,
+  RecentResource,
+  Workspace,
+  WorkspaceAccessLevel
+} from 'generated/fetch';
+import {CohortListResponse} from 'generated/fetch/api';
+import {WorkspaceStubVariables} from './workspace-service-stub';
+
+export let DEFAULT_COHORT_ID = 1;
+export let DEFAULT_COHORT_ID_2 = 2;
+
+class CohortStub implements Cohort {
```

Comment:
Why is this class needed? Can you just directly instantiate an object as needed which matches the interface type?

Reply:
Not addressed